src/main/java/org/jenkinsci/plugins/GithubAuthenticationToken.java

```diff
@@ -457,14 +478,17 @@ public GHUser loadUser(String username) throws IOException {
         return user != null ? user.user : null;
     }
 
-    public GHMyself loadMyself(String token) throws IOException {
+    private GHMyself loadMyself(@Nonnull String token) throws IOException {
         GithubMyself me;
         try {
             me = usersByTokenCache.getIfPresent(token);
             if (me == null) {
                 GHMyself ghMyself = getGitHub().getMyself();
                 me = new GithubMyself(ghMyself);
                 usersByTokenCache.put(token, me);
+                // Also stick into usersByIdCache (to have latest copy)
+                String username = ghMyself.getLogin();
+                usersByIdCache.put(username, new GithubUser(ghMyself));
```

Comment:
is it possible this code is throwing an exception that is causing https://issues.jenkins-ci.org/browse/JENKINS-56997
It seems like this method is returning `UNKNOWN_TOKEN`. I'm running into this issue when scanning my organization for a pipeline configuration if that helps.

Reply:
Thanks for the note I’ll investigate this